catalog/spatial/geocoding/spatial-geocoding-api/src/main/java/org/codice/ddf/spatial/geocoding/GeoCodingConstants.java

```diff
@@ -39,4 +42,15 @@
   public static final String SEAT_FOURTH_ORDER = "A4";
 
   public static final String CAPITAL = "C";
+
+  public static final String GEOCODER_METACARD_TAG = "gazetteer";
+
+  // The GeoNames feature codes for cities, excluding cities that no longer exist or that have
+  // been destroyed.
```

Comment:
Are all these codes for non-existent cities?  Are there many?  Do we need to exclude them?  If I want to search on ephesus, it's still a location.  If we do want to remove them, can we rename this variable... with its current name,at a glance I thought we are removing all city data.

Reply:
These are the feature codes for existent cities.  We omit the codes for non-existent cities.